src/libPMacc/PMaccConfig.cmake

```diff
@@ -29,7 +29,7 @@
 ###############################################################################
 # PMacc
 ###############################################################################
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Why could you downgrade this requirement?

Please add your name to the author list.